pkg/config/initoptions.go

```diff
@@ -148,16 +154,24 @@ func allInSameNamespace(packageDir string) (string, bool, error) {
 		if err != nil {
 			return "", false, err
 		}
+		// Skip cluster-scoped resources
+		namespaced, found := openapi.IsNamespaceScoped(rm.TypeMeta)
```

Comment:
We should just add a comment here that if we can't determine is a resource is namespace-scoped, we will default to assuming it is namespaced. That sounds reasonable. Also we should make sure this is reflected in the docs.

Reply:
Updated comment. Docs will be updated in separate PR.